### DISCUSSION TITLE
Copilot writes disturbing things in Latin

### DISCUSSION THREAD

**Discussion by mathitoledo-yo** [community/community#15652] [Copilot Conversations]
I am a student of react in Spanish and I found the possibility of using copilot, as I was doing the course I found good translations of copilot’s part, but now that I am doing my first project I start writing pretty weird pre-defined things, and just create a page with a nav bar and three sheets in javascript.
Such things as...

```
const Homer = () => {
    return (
        <div>
        <h1>Homer</h1>
        <p>
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Quibusdam
            quaerat, doloremque, dolore quas, voluptate velit, quisquam,
            necessitatibus quos dolores quae quidem dolorum.
        </p>
        </div>
    );
    }

    export default Homer;
    ```
your translation would be something like:
The pain itself is important to the main adipisicing elite. To some, let him ask, let him drive back, by the pain which he wants, by the pain, by the pain, by the necessities.
Can someone tell Copilot to stop philosophizing and get to work? if I wanted a hippie I would go to the university of social sciences

**Comment by davecheney** [comment]
I'm sorry you experienced this. Lorem ipsum is [standard placeholder boilerplate that is used in copywriting](https://en.wikipedia.org/wiki/Lorem_ipsum), it's almost a meme. Could you explain a little more what you did, and what copilot did in response?

**Comment by B0GE** [comment]
This right here is literaly digital gold.
